src/modules/src/controller_lee_payload.c

```diff
@@ -445,6 +367,55 @@ static void runQP(const struct QPInput *input, struct QPOutput* output)
     input->self->n6 = n6;
     // return desVirtInp;
     output->desVirtInp = desVirtInp;
+  } else {
```

Comment:
avoid just an else, use "else if", and then use an else at the very end to catch cases where we don't have any code for, yet.